src/guest/config_parser.cc

```diff
@@ -43,7 +43,7 @@ map<string_view, vector<string_view>> kConfigMap = {
     { kGroupPciPt,   { kPciPtDev }},
     { kGroupMed,     { kMedBattery, kMedThermal, kMedCamera } },
     { kGroupService, { kServTimeKeep, kServPmCtrl, kServVinput } },
-    { kGroupExtra,   { kExtraCmd, kExtraService } }
+    { kGroupExtra,   { kExtraCmd, kExtraService, kExtraPwrCtrlMultiOS } }
```

Comment:
please also complete the doc: https://github.com/projectceladon/vm_manager/blob/master/docs/fields.md#extra